src/commands/postDeploy/runPostDeployTask.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { CancellationTokenSource } from "vscode";
+import { SiteTreeItem } from "../../explorer/SiteTreeItem";
+import { checkLinuxWebAppDownDetector } from "./checkLinuxWebAppDownDetector";
+import { validateWebSite } from "./validateWebSite";
+
+export const postDeployCancelTokens: Map<string, CancellationTokenSource> = new Map();
```

Comment:
Seems like an odd place to store `postDeployCancelTokens` when you don't even use it in this file